packages/e2e-tests/helpers/authentication.ts

```diff
@@ -22,3 +22,7 @@ export const E2E_USER_3_TEAM_ID = "dzoxNzlkZGU4Mi00MmVhLTRkZTctYmI1OC04ZWY0NDQxZ
 // trunk-ignore(gitleaks/generic-api-key)
 export const E2E_USER_4_API_KEY = "rwk_CaQCN8h7Wz794K1Yx2200FOcta24j7NkJaYO07TNndV";
 export const E2E_USER_4_TEAM_ID = "dzpmMDZkNWI5OS05MWYzLTRhYTktYTYyNC0zMDRjYjJlYjBlYzk=";
+
+// With only read:workspace role
+export const TEST_RUN_WORKSPACE_API_KEY = "rwk_ONquDEbGGXrjs0yvuON1BhSiAUdlAz8CAl7BlB4igHo";
+export const TEST_RUN_WORKSPACE_TEAM_ID = "dzplODhmZjY1Mi0wYTdmLTRhZDgtOTNiYS04OTZmMTBjYjRjM2Q=";
```

Comment:
Are either of these used anywhere? I couldn't find any references.

Reply:
These are used at the start of each test.

For example:
https://github.com/replayio/devtools/blob/3382e5eb272fee205ad410731936922a2cbb148b/packages/e2e-tests/authenticated/new-tests-suite/test-runs-01.test.ts#L27